### ISSUE TITLE
Hash expansion needs removed from `**options` in call to `#pattern_from`.

### ISSUE THREAD

**Issue by brett-novu** [sinatra/mustermann#95]
https://github.com/sinatra/mustermann/blob/34093954a77bbef159fa2a9fb3afc1c4b8354cda/mustermann/lib/mustermann/composite.rb#L14

This line should be:

```ruby
Mustermann[type || Mustermann::DEFAULT_TYPE].supported?(option, options)
```

**Comment by jkowens** [comment]
I'm not sure this really needs changed. If you can provide a failing spec to demonstrate that would be helpful.

**Comment by jkowens** [comment]
Ah this was probably fixed with this PR #97.
